lib/Terrain.h

```diff
@@ -1,12 +1,12 @@
 /*
- * Terrain.h, part of VCMI engine
- *
- * Authors: listed in file AUTHORS in main folder
- *
- * License: GNU General Public License v2.0 or later
- * Full text of license available in license.txt file, in main folder
- *
- */
+* Terrain.h, part of VCMI engine
```

Comment:
Disagree with that, check other files - there are single spaces everywhere in header